base/sim/fastsim/CMakeLists.txt

```diff
@@ -27,10 +27,13 @@ target_include_directories(${target} PUBLIC
 
   # TODO: DELETE ME ONCE USING root targets
   ${ROOT_INCLUDE_DIR}
+
+  ${Vmc_INCLUDE_DIR}
 )
 
 target_link_directories(${target} PUBLIC
   ${ROOT_LIBRARY_DIR}
+  ${Vmc_LIBRARY_DIR}
```

Comment:
Same here, who sets this variable? Also it is not needed.

Reply:
Removed